ci/check_diff.sh

```diff
@@ -71,6 +71,7 @@ function create_diff() {
 # $RUSFMT_BIN: Path to the rustfmt master binary. Created when running `compile_rustfmt`
 # $FEATURE_BIN: Path to the rustfmt feature binary. Created when running `compile_rustfmt`
 # $OPTIONAL_RUSTFMT_CONFIGS: Optional configs passed to the script from $4
+# $OUTPUT_DIR: Path to an output directory for storing the diff files. Set in `main`
 function check_diff() {
     echo "running rustfmt (master) on $1"
```

Comment:
Just a point of personal preference in shell scripts, but in cases where the function gets long and has repeated usage of args, I like creating function local variables and assigning the value to the corresponding positional parameter.

Otherwise one can get a few (or even dozens) of lines into a function looking positional parameter references and trying to remember which is which, or bouncing back up to the signature/docs